pkg/virt-operator/resource/apply/certificates_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2021 Red Hat, Inc.
+ *
+ */
+
+package apply
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	k8sv1 "kubevirt.io/client-go/api/v1"
+	v1 "kubevirt.io/client-go/api/v1"
+)
+
+var _ = Describe("Certificates", func() {
+	Context("With deprecated fields defined", func() {
+		two_days := &metav1.Duration{Duration: 48 * time.Hour}
```

Comment:
Not to insult your math skills ;-), but would it be somewhat simpler to have:
```go
day := &metav1.Duration{Duration: 24 * time.Hour}
twoDays := 2 * day
threeDays := 3 * day
...
```

or even:
```go
day := &metav1.Duration{Duration: 24 * time.Hour}
...
config.CA.Duration = 5 * day
```

Also note the snake_case --> camelCase.

Reply:
concur with the snake case part, but unfortunately "metav1.Duration" is not a number so we can't apply mathematical operators to it. I think you might be thinking of "time.Duration" which is a long int representing a number of milliseconds.

Reply:
Right, I overlooked the fact that `metav1.Duration` is a struct type wrapping `time.Duration` (I falsely assumed it's aliasing it...).

Could still be done with a `days(n int) metav1.Duration` helper function, but I'm not sure it's still simplifying or rather complicating. Let's leave as-in.